nautobot/extras/datasources/git.py

```diff
@@ -75,13 +88,48 @@ def pull_git_repository_and_refresh_data(repository_pk, request, job_result_pk):
         job_result.save()
         return
 
+    job_result.log(f'{log_message} "{repository_record.name}"...', logger=logger)
+    job_result.set_status(JobResultStatusChoices.STATUS_RUNNING)
+    job_result.save()
```

Comment:
This feels out-of-place for a `get_` method - it shouldn't modify the state of the `job_result`, and the `log_message` doesn't have anything directly to do with `get`ing the records either.